### PR TITLE
remove migrations

### PR DESCRIPTION
**Pull request by Rian501**
Have removed discussion of migrations and altered the sandcastle exercise to use sequelize.sync() instead, including how to set up models for join tables in order to seed them with queryInterface.bulkCreate().

Please check the md files in the sequelize directory to make sure they make sense, and also check for typos.

**As a caveat, there may be issues in using the sync method when it comes to querying join tables further down the line (that may be the trade-off for using sync) -- it can be worked around fairly easily using 'raw SQL query' methods in the project, which is probably fine, given that it gives more practice with SQL even if it does not leverage Sequelize methods to their absolute fullest. Either way, this may need to be updated as we figure that out.

### CODE REVIEW

File: post-orientation-exercises/postgres/02-intro-to-ORMs.md [hunk 1]
```diff
@@ -21,13 +21,13 @@ Then, let's escape all the pressure and stress of NSS by building some sandcastl
...
 Then `npm init` and install some dependencies
-`npm install --save sequelize`
+`npm install sequelize`
 While you're at it, throw an installation of PostgreSQL ( `pg` ) in there too. ( As of this writing, you may or may not need an additional installation of the `pg-hstore` module. Your intrepid instructors are keeping abreast of the latest from Sequelize Central. Doesn't hurt to include it for now )
 
 Launch postgres, then using psql in a new terminal window/tab, `CREATE DATABASE sandcastledb;`
 HINT: don't forget the ";" at the end of the statement, and avoid capital letters.
 
-Next, a quick `sequelize init` at the root of your project will create the basic folder/files you'll need. Your file structure should like something like this
+Next, a quick `sequelize init` at the root of your project will create the basic folder/files you'll need. The generated file structure should like something like this
```
Comment (JoeShep) [inline post-orientation-exercises/postgres/02-intro-to-ORMs.md]: "should __look__ something like this"

File: post-orientation-exercises/postgres/03-sequelize-models.md [hunk 2]
```diff
@@ -51,12 +51,13 @@ Now that `associate` has been added as a property of a Beach, what the heck is i
 
 Beach.associate = function(models) {
   Beach.hasMany(models.Lifeguard, {
-    foreignKey: 'lifeguardId'
+    foreignKey: 'beach_id' //the same foreign key must be specified when you define the association on the lifeguard model.
```
Comment (JoeShep) [inline post-orientation-exercises/postgres/03-sequelize-models.md:54]: Is this right? The foreign key needs to be `beach_id`?

File: post-orientation-exercises/postgres/03-sequelize-models.md [hunk 3]
```diff
@@ -51,12 +51,13 @@ Now that `associate` has been added as a property of a Beach, what the heck is i
...
   });
 };
 
 ...
 ```
+(When naming `foreignKey`s in your models, think of the 'hasMany' as sending the foreignKey, and the 'belongsTo' as recieving it.)
```
Comment (JoeShep) [inline post-orientation-exercises/postgres/03-sequelize-models.md]: Oh, I guess that answers my question. Weird. I guess I got that all wrong. 
Typo: "receiving"

File: post-orientation-exercises/postgres/05-sequelize-seeding.md [hunk 4]
```diff
@@ -1,62 +1,49 @@
...
-To seed your database, use `sequelize db:seed:all` to run all the seed files or `sequelize db:seed --seed seed_file_name` to seed one file. With `all`, the migrations are run by timestamps in ascending order. If you rename the files without the timestamps, you can name them in alphabetical order and they will run from a - z.
+As before, you will need a json file where the keys match exactly with the column names of your db table. Import these from wherever they live, then you can bulkCreate them right into the tables.
+
+```
+let sequelize = require('sequelize');
+let queryInterface = require('sequelize/lib/query-interface');
+
+const { users } = require('./data/users.json')
+const { Computers } = require('./data/computers.json')
+
```
Comment (JoeShep) [inline post-orientation-exercises/postgres/05-sequelize-seeding.md]: One of these destructured props is caps and the other isn't. Let's make them match

File: post-orientation-exercises/postgres/05-sequelize-seeding.md [hunk 5]
```diff
@@ -1,62 +1,49 @@
...
+let createdb = (queryInterface) => {
+  const app = require('./app');
+  const models = app.get('models');
```
Comment (JoeShep) [inline post-orientation-exercises/postgres/05-sequelize-seeding.md]: `app.set` will have to have happened in app.js for this to work. Is that explained anywhere?

File: post-orientation-exercises/postgres/06-sequelize-relationships.md [hunk 6]
```diff
@@ -26,59 +26,21 @@ Show.associate = (models) => {
...
-          key: 'id'
-        },
-        onUpdate: 'cascade',
-        onDelete: 'cascade'
-      },
-      // these are optional, but you have to tell Sequelize you don't want them in the model. See the docs
-      createdAt: {
-          allowNull: false,
-          type: Sequelize.DATE
+If we want to allow duplicates of the same realtionship, we'll also need to specify `unique: false` and `constraints: false`. This isn't useful with users and favorite shows, but it might come in handy with other things later...
```
Comment (JoeShep) [inline post-orientation-exercises/postgres/06-sequelize-relationships.md]: "relationship"

**Review by JoeShep — APPROVED** [review]
Looking good! I'm sure some weirdness will present itself, but this gets us started down the `synch` path. Thanks for slogging thru it 👍

**Comment by JoeShep** [comment]
Approved pending changes noted in comments

File: post-orientation-exercises/postgres/05-sequelize-seeding.md (hunk 5 above)
Comment (Rian501) [reply post-orientation-exercises/postgres/05-sequelize-seeding.md]: I think it ends up being part of the pug file setup that they learn before this stuff, but I think I should include a note referencing it either way. Then if they are unclear they at least have something to target when asking for help. 👍

File: post-orientation-exercises/postgres/03-sequelize-models.md (hunk 3 above)
Comment (Rian501) [reply post-orientation-exercises/postgres/03-sequelize-models.md]: That's not a typo, that's how I spell the r-word that means 'to accept something given or sent' ✉️

**Pull request merged**